### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by tribal-tec** [comment]
Retest this please

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1 @@
 .travis.yml merge=ours
-CMake/FindPackages.cmake merge=ours
-CMake/FindRequired.cmake merge=ours
-CMake/GitExternal.cmake merge=ours
-CMake/depends.txt merge=ours
```
Comment (eile) [inline .gitattributes]: hmm, dragons.

**Comment by tribal-tec** [comment]
clang-format drives me nuts though. No idea what the issues is

File: vmmlib/enable_if.hpp [hunk 2]
```diff
@@ -41,12 +41,17 @@
  */
 namespace vmml
```
Comment (hernando) [inline vmmlib/enable_if.hpp]: Is this file needed?

Comment (tribal-tec) [inline vmmlib/enable_if.hpp]: probably not anymore. It's c&p from boost enable_if, but now we should use C++11. Good catch!

**Comment by tribal-tec** [comment]
clang-format seems to be fine. Thx @hernando
